### DISCUSSION TITLE
UGS with Arduino Uno

### DISCUSSION THREAD

**Discussion by Mendelugs** [winder/Universal-G-Code-Sender#2445] [General]
Have try all procedures explained on tutorial help page and never getting connect. What should be the issue?.

**Comment by breiler** [comment]
Without details it will be impossible to answer.

Which tutorial?
Which version of UGS are you using?
What operating system are you using?
Do you get any error messages?
Which firmware and version have you loaded to your Uno?

**Reply by Mendelugs** [reply]
I read the ugs help window, I am running Mojave on my MacBook Pro, and Window 10 on my Dell Laptop. Both are running the latest version on both Arduino Uno and UGS. Thanks

**Reply by breiler** [reply]
Sorry, I won't be able to help without details.

Which GRBL did you upload to the Arduino?   Have you verified that GRBL is working on your Arduino?

**Reply by Mendelugs** [reply]
I am using latest version on both. the Arduino software is detecting the Arduino Uno card. The TIF is being detected on Arduino, The Grbl was present before TIF. But I have tried re installing the Arduino software and no success. Thanks for your help.

**Reply by Mendelugs** [reply]
Okay good news to you from me. I successfully got connect on my dell computer but not yet on any of my Macs and found the Grbl library is not being detected but the TFT. I have to find out what happened. If some one should help me with this issue I will appreciate it. Thanks
